doc/user/content/ingest-data/mysql/amazon-rds.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Ingest data from Amazon RDS for MySQL"
+description: "How to stream data from Amazon RDS for MySQL to Materialize"
+menu:
+  main:
+    parent: "mysql"
+    name: "Amazon RDS"
+    identifier: "mysql-amazon-rds"
+---
+
+{{< private-preview />}}
+
+This page shows you how to stream data from [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql/)
+to Materialize using the [MySQL source](/sql/create-source/mysql).
+
+## Before you begin
+
+{{% mysql-direct/before-you-begin %}}
+
+## Step 1. Enable GTID-based binlog replication
+
+Before creating a source in Materialize, you **must** configure Amazon RDS for
+GTID-based binlog replication. This requires the following configuration
+changes:
+
+Configuration parameter          | Value  | Details
+---------------------------------|--------| -------------------------------
```

Comment:
@rjobanp: We were also specifying `log-slave-updates` (resp. `log_replica_updates`) in our tests. Should this parameter be mentioned here?

https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_log_replica_updates